hub/core/chunk_engine/write.py

```diff
@@ -1,69 +1,70 @@
+from hub.constants import UNCOMPRESSED, USE_UNIFORM_COMPRESSION_PER_SAMPLE
+from hub.core.sample import Sample
```

Comment:
We're gonna have a ton of conflicts in this file with transforms

Reply:
that's what i was saying

Reply:
we can do them together though